fetch.bs

```diff
@@ -2971,16 +2972,18 @@ boolean <var>http3Only</var>, run these steps:
   [[!HTTP1]] [[!TLS]]
 
   <ul>
-   <li><p>If <var>http3Only</var> is true, then establish an HTTP/3 connection. [[!HTTP3]]
+   <li>
+    <p>If <var>requireUnreliable</var> is true, then establish a connection capable of unreliable
+    transport, e.g., HTTP/3. [[!HTTP3]]
 
-   <li><p>When establishing an HTTP/3 connection, include SETTINGS_ENABLE_WEBTRANSPORT with a value
-   of 1 and H3_DATAGRAM with a value of 1 in the initial SETTINGS frame. [[!WEBTRANSPORT-HTTP3]]
-   [[!HTTP3-DATAGRAM]]
```

Comment:
See https://github.com/whatwg/fetch/pull/1745#discussion_r1578222835 on a resolved thread.

Reply:
@jan-ivar I see, I guess then it should say "When establishing a connection capable of unreliable transport, enable options ..." I wonder if we have any cases where it's false and this is still relevant?

Reply:
@annevk It's likely false almost always. The WebTransport constructor's [requireUnreliable](https://w3c.github.io/webtransport/#dom-webtransportoptions-requireunreliable) option merely exists to disable HTTP/2 fallback, but likely nobody sets it since browsers still need to implement HTTP/2 fallback.

There's also [pooling](https://w3c.github.io/webtransport/#dom-webtransportoptions-allowpooling) to consider aka `!dedicated` (also not implemented yet AFAIK) where the WebTransport constructor is returned an existing connection from this algorithm's connection pool. 

Together this means both SETTINGS_ENABLE_WEBTRANSPORT and H3_DATAGRAM need to be enabled on _all_ HTTP/3 connections by default, even when not called from WebTransport. At least that's how I read it.

If this is surprising to others (or if I read this wrong), we might want to make this clearer, as I could see someone reading it the other way right now.